packages/builder/src/pages/builder/portal/apps/[appId]/index.svelte

```diff
@@ -45,14 +60,26 @@
       </ActionButton>
     {/if}
     <ActionButton
+      disabled={noScreens}
       quiet
       icon="LinkOut"
       on:click={() => window.open(iframeUrl, "_blank")}
     >
       Fullscreen
     </ActionButton>
   </div>
-  <iframe src={iframeUrl} title={app.name} />
+  {#if noScreens}
+    <div class="noScreens">
+      <ErrorSVG />
+      <Body>You haven't added any screens to your app yet.</Body>
+      <Body
+        ><a href={$url(`/builder/app/${app.devId}/design`)}>Click here</a> to add
```

Comment:
We should use the `Link` component here to keep consistent with the rest of the platform. That will also style the link blue rather than indigo, which I think is the safer choice until we properly move our accent color over.